lib/ui/home/HomeDailyIlliniWidget.dart

```diff
@@ -342,14 +345,19 @@ class _MainStoryWidget extends StatelessWidget {
       child: Column(
         crossAxisAlignment: CrossAxisAlignment.start,
         children: [
-          ClipRRect(
-              borderRadius: BorderRadius.vertical(top: Radius.circular(4)),
-              child: _buildImage()
-          ),
+          InkWell(
+            onTap: () => launchUrlString(StringUtils.ensureNotEmpty(illiniItem?.link)),
```

Comment:
This opens black screen in Android devices. Please use the following appraoch.

```
  void _onTap() {
    String? url = illiniItem!.link;
    if (StringUtils.isNotEmpty(url)) {
      Uri? uri = Uri.tryParse(url!);
      if (uri != null) {
        LaunchMode launchMode = Platform.isAndroid ? LaunchMode.externalApplication : LaunchMode.platformDefault;
        launchUrl(uri, mode: launchMode);
      }
    }
  }
```

This is tested and working.

Reply:
fixed